infra/build_specified_commit.py

```diff
@@ -22,88 +22,98 @@
 import subprocess
 
 import helper
-import repo_manager
 
 
-class DockerExecutionError(Exception):
-  """An error that occurs when running a docker command."""
+class BuildData():
+  """List of data requried for bisection of errors in OSS-Fuzz projects.
+
+  Attributes:
+    project_name: The name of the OSS-Fuzz project that is being checked.
+    engine: The fuzzing engine to be used.
+    sanitizer: The sanitizer to be used.
+    architecture: CPU architecture to build the fuzzer for.
+  """
+  # pylint: disable=too-few-public-methods
```

Comment:
If we're not using a Dataclass I think these attributes need to be set in the constructor. Otherwise they are part of the class and not the object?

Reply:
Done.

Reply:
I meant make them object variables (as they are now with my edits), not class variables